### PR TITLE
[ZT] CASB + DLP

### PR DESCRIPTION
**Pull request by ranbel**
PCX-5536

### CODE REVIEW

**Comment by ranbel** [comment]
Need to add redirects for DLP pages.

File: content/cloudflare-one/applications/scan-apps/casb-dlp.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+# Scan SaaS applications with DLP
+
+{{<Aside type="note">}}
+
+Requires Cloudflare CASB and Cloudflare DLP.
+
+{{</Aside>}}
+
+You can use [Cloudflare Data Loss Prevention (DLP)](/cloudflare-one/policies/data-loss-prevention/) to discover if files stored in your SaaS application contain sensitive data.
+To perform DLP scans in a SaaS app, first configure a DLP profile with the data patterns you want to detect, and then enable those profiles in a CASB integration.
```
Comment (maxvp) [inline content/cloudflare-one/applications/scan-apps/casb-dlp.md]: Nit: Is the `and` before `then` necessary?

File: content/cloudflare-one/applications/scan-apps/casb-dlp.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+## Configure a DLP profile
+
+Refer to the [DLP documentation](/cloudflare-one/policies/data-loss-prevention/dlp-profiles/).
+
+## Enable DLP scans in CASB
+
+### Add a new integration
+
+1. In the [Zero Trust dashboard](https://dash.teams.cloudflare.com), go to **CASB** > **Integrations**.
```
Comment (maxvp) [inline content/cloudflare-one/applications/scan-apps/casb-dlp.md]: Do we use `https://dash.teams.cloudflare.com` or `https://one.dash.cloudflare.com/`?

File: content/cloudflare-one/policies/data-loss-prevention/dlp-policies/common-policies.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+The **Allow** action functions as an implicit logger, providing visibility into where your sensitive data is going without impacting the end user experience. The following example scans for your enabled Financial Information profile entries when users upload or download data to file sharing apps.
+
+| Policy name |
+| ---- |
+| Log Financial Information file sharing|
+
+| Selector | Operator | Value |
+| - | - | - |
+| DLP Profile | in | `Financial Information` |
```
Comment (maxvp) [inline content/cloudflare-one/policies/data-loss-prevention/dlp-policies/common-policies.md:22]: With the upcoming OR operator in the policy builder, will we need to explicit state that policies will need an AND between conditions?

**Review by maxvp — APPROVED** [review]

File: content/cloudflare-one/applications/scan-apps/casb-dlp.md (hunk 1 above)
Comment (ranbel) [inline content/cloudflare-one/applications/scan-apps/casb-dlp.md]:
```suggestion
To perform DLP scans in a SaaS app, first configure a DLP profile with the data patterns you want to detect, then enable those profiles in a CASB integration.
```

File: content/cloudflare-one/applications/scan-apps/casb-dlp.md (hunk 2 above)
Comment (ranbel) [reply content/cloudflare-one/applications/scan-apps/casb-dlp.md]: The old link redirects to the new one, but we should eventually do a find-and-replace in the docs.

Comment (ranbel) [inline content/cloudflare-one/applications/scan-apps/casb-dlp.md]:
```suggestion
1. In the [Zero Trust dashboard](https://one.dash.cloudflare.com/), go to **CASB** > **Integrations**.
```

**Pull request merged**